app/src/test/kotlin/compiler/e2e/LexicallyIncorrectPrograms.kt

```diff
@@ -0,0 +1,78 @@
+package compiler.e2e
+
+import compiler.common.diagnostics.Diagnostic
+import compiler.lexer.Location
+import kotlin.test.Test
+
+class LexicallyIncorrectPrograms {
```

Comment:
These test fail because the incorrect code from Lexer causes further errors in Parser.